CONTRIBUTING.md

```diff
@@ -18,7 +18,7 @@ Setup guides, FAQ, the more information we have on the wiki the better.
 1.  Fork Sonarr
 2.  Clone (develop branch) *you may need pull in submodules separately if you client doesn't clone them automatically (CurlSharp)*
 3.  Run `npm install`
-4.  Run `gulp watch` - Used to compile the UI components and copy them (leave this window open)
+4.  Run `npm run gulp watch` - Used to compile the UI components and copy them (leave this window open)
```

Comment:
tbh, I think most people that would be contributing might be confused by this, specially if they are already familiar with gulp, since most people would know gulp as using it by calling the globally installed version.